server/package.json

```diff
@@ -1,24 +1,26 @@
 {
```

Comment:
nit: curious what happened with the formatting here. I was using two-spaces indentation, can we keep it this way? :)